readme.md

```diff
@@ -692,7 +692,7 @@ If you want to predefine selected values, give the choice object an `selected` p
 > Interactive auto complete prompt.
 
 The prompt will list options based on user input. Type to filter the list.
-Use <kbd>up</kbd>/<kbd>down</kbd> to navigate. Use <kbd>tab</kbd> to cycle the result. Use <kbd>Page Up</kbd>/<kbd>Page Down</kbd> to change page. Hit <kbd>enter</kbd> to select the highlighted item below the prompt.
+Use <kbd>🠹</kbd>/<kbd>🠻</kbd> to navigate. Use <kbd>tab</kbd> to cycle the result. Use <kbd>Page Up</kbd>/<kbd>Page Down</kbd> (on Mac: <kbd>fn</kbd> + <kbd>🠹</kbd> / <kbd>🠻</kbd>) to change page. Hit <kbd>enter</kbd> to select the highlighted item below the prompt.
```

Comment:
for some reason the these characters aren't rendering for me (on a mac)

![Screen Shot 2019-03-09 at 3 12 13 AM](https://user-images.githubusercontent.com/2242/54068285-3c9c4280-4219-11e9-9209-7d411fdc5a76.png)

Reply:
It's cause IOS still use old unicode version.

I fixed it. You should see the characters now